### ISSUE TITLE
Bug on new item creation: Item X is linked with existing records

### ISSUE THREAD

**Issue by bynicolas** [frappe/books#175]
There is a relation between `incomeAccount` and `expenseAccount` that must be made in regards to `items`.

On the **Items** screen's **QuickEditForm**, the field for `incomeAccount` and `expenseAccount` are not showing so you cannot link your new item to the correct chart account for those attributes.

My chart of accounts has non english account name, if that can help pin point the issue.

A similar behaviour is shown in #130, but in that particular case, a field is available to choose a correct account value.

**Comment by ccrvlh** [comment]
+1 same here.

**Comment by madeadi** [comment]
+1 same here

**Comment by Luigi-** [comment]
+1 same here using version 0.0.3-beta.13

**Comment by tlccstudio** [comment]
+1 same here, makes the product not usable on linux ubuntu 18.   Tried many things to make it work.  
![image](https://user-images.githubusercontent.com/77962211/117035607-b4e5ec00-ad2e-11eb-85a8-3572d2d0d8cb.png)
![image](https://user-images.githubusercontent.com/77962211/117035625-bc0cfa00-ad2e-11eb-8883-3c0640f12f9f.png)

**Comment by oaeide** [comment]
Same :(

**Comment by 18alantom** [comment]
@bynicolas, @oaeide can you provide steps to reproduce? Unable to replicate it on _v0.0.3-beta.13_. or `master` when the country is set to India (Engilish).

A few things that will help in debugging this issue:
- Steps to reproduce
- Language
- Country
- Item name
- Your db file itself (if there isn't any personal info on it)

**Comment by wouter-deen** [comment]
I'm also having this issue. I'm from the Netherlands and I tried it first with the Netherlands country preset. Then, I deleted my db file, I re-created my company in the app, and selected United States instead. This still gave me the same problem, when doing almost anything. I can't even book any expenses or income. See screenshots below:

![image](https://user-images.githubusercontent.com/62772323/139130302-fb30619c-4898-42bf-8ac6-73175cf01f87.png)
Clicking save results in this popup:
![image](https://user-images.githubusercontent.com/62772323/139130360-fde4d38d-9fe7-46ca-818e-723a6e5c5841.png)
